packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php

```diff
@@ -10,16 +10,16 @@
 /**
  * This class allows us to define multiple search strategy for the same property on a single resource.
  *
- * The default behaviour of SearchFilter is preserved + ability to define "virtual properties" mapped to real ones:
+ * The default behaviour of SearchFilter is preserved + ability to define "virtual properties" mapped to real ones:.
  *
  * ApiFilter(
  *     VirtualSearchFilter::class,
- *     properties={
- *         "number_partial": {"number": "ipartial"}, --> Virtual search property to number partial
- *         "number_exact": {"number": "exact"}, --> Virtual search property to number exact
- *         "email": "ipartial" --> Normal search property definition
- *     }
- * )
+ *     properties={.
```

Comment:
@TomasVotruba what about those dots? 😄

Reply:
You should not write broken rules :D  I'll check it.

Reply:
Is that us!? 😄 We can get someone to check then

Reply:
I'll handle it :)

I need someone to merge my other PRs here, so it does not stack/stuck

Reply:
Yes no worries, I will merge everything today